osm2streets/src/operations/collapse_short_road.rs

```diff
@@ -26,36 +26,18 @@ impl StreetNetwork {
             (r.src_i, r.dst_i)
         };
 
-        // If either intersection attached to this road has been deleted, then we're probably
```

Comment:
This check is no longer needed. We should never have a road that refers to non-existent intersections, ever. If we do, something below will crash, and we can track down the problem manually. (Maybe with that consistency check idea)